integration-tests/runtime-tests/test/tests/dexTests/constantProductDex/constantProductTests.ts

```diff
@@ -0,0 +1,110 @@
+import testConfiguration from './test_configuration.json';
+import {expect} from "chai";
+import {KeyringPair} from "@polkadot/keyring/types";
+import { addFundstoThePool, buyFromPool, createPool, mintAssets, removeLiquidityFromPool, sellToPool, swapTokenPairs } from './testHandlers/constantProductDex';
+
+/**
+   * This suite includes tests for the constantProductDex Pallet. 
+   * Tested functionalities are:
+   * Create - AddLiquidity - Buy - Sell - Swap - RemoveLiquidity with basic calculations with constantProductFormula.  
+   * Mainly consists of happy path testing.
+*/
+describe('tx.constantProductDex Tests', function () {
+
+  let walletId1:KeyringPair,
+  walletId2:KeyringPair,
+  sudoKey:KeyringPair;
+  let poolId:number,
+  baseAssetId:number,
+  quoteAssetId:number,
+  aliceLpTokens:number,
+  baseAmount:number,
+  quoteAmount:number;
+    
+  before('Initialize variables', function() {
+    walletId1=walletAlice;
```

Comment:
For best practice, derive a new wallet from our set of base wallets.
I like to use the path `/test/testGroupName/testWalletName` for this.
In this case something like:
`walletId1 = walletAlice.derive("/test/constantProductDex/wallet1")`
(afaik this only works for non-sudo calls since sudo calls require the alice main wallet.)

Reply:
Done, thanks for the suggestion. Implemented now.